lib/ReactViews/FeatureInfo/FeatureInfoPanel.jsx

```diff
@@ -68,6 +80,11 @@ const FeatureInfoPanel = createReactClass({
         }
     },
 
+    checkDragging() {
+        console.log(this.ref.current.node.getAttribute('data-is-dragging'));
```

Comment:
ohh I see what `data-is-dragging` is used for now. 
Two minor things: remove the `console.log`, and consider renaming this function to `isDragging` cause that makes it clearer what the true/false means (not that it's hard to figure out).